src/webgpu/api/validation/encoding/cmds/render/state_tracking.spec.ts

```diff
@@ -74,6 +74,20 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
+g.test(`all_needed_buffer_should_be_bound`)
+  .desc(
+    `
+In this test we test that any missing buffer for a used slot will cause validation errors when drawing.
+- All (non/indexed, in/direct) draw commands
+    - A needed vertex buffer is not bound
+        - Was bound in another render pass but not the current one
+        - x= all vertex formats
+- Indexed draw commands,
+    - No index buffer is bound
```

Comment:
Should almost certainly be a separate test.

Reply:
Done.